application/admin/views.py

```diff
@@ -102,7 +103,10 @@ def add_user():
         db.session.commit()
         create_and_send_activation_email(form.email.data, current_app)
         return redirect(url_for("admin.users"))
-    return render_template("admin/add_user.html", form=form)
+
+    return render_template(
+        "admin/add_user.html", form=form, error_summary=get_error_summary_data(title="There is a problem", forms=[form])
```

Comment:
Did this make it past content design? :stuck_out_tongue:

Reply:
It's the exact content recommended by the GOV.UK Design System, so it has been content designed already.

See: https://design-system.service.gov.uk/components/error-summary/#how-it-works

* `include the heading ‘There is a problem’`